mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/MavenPropertyResolver.java

```diff
@@ -18,4 +23,13 @@ static private boolean isSkip() {
         return Boolean.valueOf(System.getProperty("maven.test.skip", "false"));
     }
 
+    static boolean isSpecificTestClassSet() {
+        String specificTestClasses = System.getProperty("test");
+        return specificTestClasses != null && !containsPattern(specificTestClasses);
+    }
+
+    private static boolean containsPattern(String specificTestClasses) {
+        Matcher matcher = TEST_CLASS_PATTERN.matcher(specificTestClasses);
```

Comment:
And probably I can update the documentation that if the user wants to run specific multiple tests irrespective of what is configured with ST that can be done by explicitly disabling smart-testing.